rllib/models/tests/test_convtranspose2d_stack.py

```diff
@@ -0,0 +1,53 @@
+import cv2
+import numpy as np
+import unittest
+
+from ray.rllib.models.torch.modules.convtranspose2d_stack import \
+    ConvTranspose2DStack
+from ray.rllib.utils.framework import try_import_torch, try_import_tf
+
+torch, nn = try_import_torch()
+tf1, tf, tfv = try_import_tf()
+
+
+class TestConvTranspose2DStack(unittest.TestCase):
+    """Tests our ConvTranspose2D Stack modules/layers."""
+
+    def test_convtranspose2d_stack(self):
+        """Tests, whether the conv2d stack can be trained to predict an image.
+        """
+        batch_size = 128
+        input_size = 1
+        module = ConvTranspose2DStack(input_size=input_size)
+        optim = torch.optim.Adam(module.parameters(), lr=0.0001)
+        img = cv2.imread("../../tests/data/images/obstacle_tower.png")
+        # Make channels first.
+        img = np.transpose(img, (2, 0, 1))
+        # Add batch rank and repeat.
+        imgs = np.reshape(img, (1, ) + img.shape)
+        imgs = np.repeat(imgs, batch_size, axis=0)
+        # Normalize (-1.0 to 1.0).
+        imgs = (imgs / 255.0) * 2.0 - 1.0
+        imgs = torch.from_numpy(imgs)
+        init_loss = loss = None
+        for _ in range(10):
+            # Random inputs.
+            inputs = torch.from_numpy(
+                np.random.normal(0.0, 1.0, (batch_size, input_size))).float()
+            distribution = module(inputs)
+            # Construct a loss.
+            loss = -torch.mean(distribution.log_prob(imgs))
```

Comment:
Are there preexisting agents that use conv transpose stack?

Reply:
Then you could probably simplify this code

Reply:
Dreamer, no? :) Other than that, nothing yet, to my knowledge. I just made up something quick here to test the model + some loss.

Reply:
Aight wfm